doc/source/examples/sharp-immersed-boundary/3d-rbf-static-mixer/3d-rbf-static-mixer.rst

```diff
@@ -0,0 +1,293 @@
+#. The mesh range of ``0.1`` means that there is at least 10% of margin on each side of the object, so the collection of RBF nodes are sure to encompass the whole object.
+
+.. code-block:: text
+
+    nb_subdivision nb_adaptions radius_ratio base_function mesh_range
+    16             4            3            1             0.1
+
+From the ``/rbf_generation/`` directory, we can launch RBF generation using the command line:
+
+.. code-block:: text
+  :class: copy-button
+
+  ./RBF_example_00001 ./ helix RBF.param
+
+After a few minutes this executable will output ``RBF_helix.output``, which is the encoding of the shape, and ``RBF_helix.vtu``, which can be used to see the resulting approximation.
+
+Creation of the Composite Shape File
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The complete geometry through which the flow happens contains the helix static mixer as well as the casing around it. We use composite shapes to build the complex geometry; this type of shape is introduced in this example: :doc:`../geometry-definition`. The main particularities of this composite shape are:
+
+#. The translation parameter for the ``rbf`` shape is ``-76.201:-20.0098:+15.6051``. It is selected to ensure that the center of the static mixer is located at the origin. The coordinates are taken from ``rbf_generation/bitpit.log``.
+#. The ``hyper rectangle`` is long enough to cover the length of the helix, and just large enough to fit in the background grid.
+#. The ``cylinder`` hole is set to have a very high length to ensure that the difference operation applies properly over the whole domain.
+#. Operation ``15`` forms the casing, and operation ``16`` joins the casing and the helix. The final operation is the one considered as definitive.
+
+.. code-block:: text
+
+    shapes
+    0; rbf             ; RBF_helix.output;   -76.201:-20.0098:+15.6051;    0:+1.57079632679:0
+    1; hyper rectangle ;         75:25:25;                       0:0:0;                 0:0:0
+    2; cylinder        ;         15:10000;                       0:0:0;    0:+1.57079632679:0
+    operations
+    15; difference     ; 2:1
+    16; union          ; 0:15
+
+---------------
+Parameter File
+---------------
+
+Simulation Control
+~~~~~~~~~~~~~~~~~~
+
+Although we are interested in the steady state solution of the flow, we use ``bdf1`` time integration. The required time to reach steady state in our case is low, but resolving it with low time steps enables the non-linear solver to converge; complex flow patterns are difficult to capture.
+
+.. code-block:: text
+
+    subsection simulation control
+      set method      = bdf1
+      set time end    = 40e-4
+      set time step   = 1e-4
+      set output path = ./output/
+      set output name = output
+    end
+
+Physical Properties
+~~~~~~~~~~~~~~~~~~~
+
+We assume that the used fluid is water, and that the static mixer is of the meter length scale (150 centimeters). The time unit is seconds and the length unit is centimeter.
```

Comment:
I don't really understand the meter length scale detail... maybe you can rewrite it to make it clearer?